dist/udb3-angular.css

```diff
@@ -6296,6 +6299,9 @@ img.offer-image-main {
 .place-detail .media-object {
   max-height: 100px;
 }
+.alert-info.lesson-series-info {
```

Comment:
Double css is generated.
Maybe you can remove the css in one file? 
Or move it to the a more general place?